.gitignore

```diff
@@ -1,4 +1,6 @@
+.idea
```

Comment:
Files related to your local dev environment should be added in your global gitignore instead of per project basis.